src/Adapter/Product/Image/Update/ProductImageUpdater.php

```diff
@@ -81,13 +90,15 @@ public function __construct(
         ProductImageUploader $productImageUploader,
         PositionUpdateFactory $positionUpdateFactory,
         PositionDefinition $positionDefinition,
-        GridPositionUpdaterInterface $positionUpdater
+        GridPositionUpdaterInterface $positionUpdater,
+        ProductImageMultiShopRepository $productImageMultiShopRepository
```

Comment:
We shouldn't have both ProductImageMultiShopRepository and ProductImageRepository, ProductImageMultiShopRepository should totally replace the other one so that we anticipate the merge of both later and it forces us to correctly use the method adapted to multishop everywhere